tests/integration_tests/performance/test_block_ab.py

```diff
@@ -240,3 +247,104 @@ def test_block_vhost_user_performance(
             **vm.dimensions,
         }
     )
+
+
+def _check_block_size(ssh_connection, dev_path, size):
+    """
+    Checks the size of the block device.
+    """
+    _, stdout, stderr = ssh_connection.run("blockdev --getsize64 {}".format(dev_path))
+    assert stderr == ""
+    assert stdout.strip() == str(size)
+
+
+@pytest.mark.nonci
+@pytest.mark.parametrize("vcpu_count", [1, 2], ids=["1vcpu", "2vcpu"])
+def test_vhost_user_block(microvm_factory, guest_kernel, rootfs, vcpu_count, metrics):
+    """
+    This test simply tries to boot a VM with vhost-user-block
+    as a root device and upload FirecrackerMetrics emitted by
```

Comment:
we don't use vhost-user-blk as a root here, we use it as a scratch drive instead.